src/pages/SupplierInvoicePage.js

```diff
@@ -431,11 +432,9 @@ export class SupplierInvoicePage extends React.Component {
 }
 
 export function checkForFinaliseError(transaction) {
-  if (transaction.items.length === 0) {
-    return modalStrings.add_at_least_one_item_before_finalising;
-  } else if (transaction.totalQuantity === 0) {
-    return modalStrings.stock_quantity_greater_then_zero;
-  }
+  if (!transaction.isExternalSupplierInvoice) return null;
```

Comment:
You'd hope, but it can be done. Just did it. mSupply doesn't validate what it sends out.